internal/flink/command_application_webui.go

```diff
@@ -6,9 +6,12 @@ import (
 	"io"
 	"net"
 	"net/http"
+	"os"
 
 	"github.com/spf13/cobra"
 
+	"github.com/confluentinc/cli/v4/pkg/auth"
+	perrors "github.com/confluentinc/cli/v4/pkg/errors"
```

Comment:
Nitpick: We probably don't need to use `perrors` here since we don't have another package or variable named `errors`.